test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -88,3 +88,31 @@ Describe "Import-Module for Binary Modules in GAC" -Tags 'CI' {
         }
     }
 }
+
+Describe "Import-Module for Binary Modules" -Tags 'CI' {
+
+    It "PS should try to load the assembly from file path first" {
```

Comment:
Are you sure this test validates the fix? It passes even without your fix.

Reply:
previously when system.automation.management is used loading error will be occured. 
Now we have to check the actual cmdlet is loaded.
Resolved.